src/Plugin/Woocommerce/Checkout.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace AdCaptcha\Plugin\Woocommerce\Checkout;
+
+use AdCaptcha\Widget\AdCaptcha\AdCaptcha;
+use AdCaptcha\Widget\Verify\Verify;
+use AdCaptcha\AdCaptchaPlugin\AdCaptchaPlugin;
```

Comment:
This will need to have modification after when the wordpress composer autoload branch  will be merged in.

Reply:
@vasswann Yes, your branches will need to be updated.